src/Features/Traits/Notification.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Features\Traits;
+
+use Blumilk\BLT\Helpers\RecognizeClassHelper;
+use Illuminate\Contracts\Container\BindingResolutionException;
+use Illuminate\Notifications\ChannelManager;
+use Illuminate\Support\Testing\Fakes\NotificationFake;
+
+trait Notification
+{
+    use Application;
+
+    public NotificationFake $notificationFake;
+
+    /**
+     * @Given notifications are being faked
+     * @throws BindingResolutionException
+     */
+    public function fakeNotifications(): void
+    {
+        $this->notificationFake = $this->getContainer()->make(NotificationFake::class);
+        $this->getContainer()->instance(ChannelManager::class, $this->notificationFake);
```

Comment:
Please use `\Illuminate\Contracts\Notifications\Dispatcher` contract instead of `ChannelManager`.